cardano-testnet/test/cardano-testnet-test/cardano-testnet-test.hs

```diff
@@ -75,27 +78,27 @@ tests = do
           , H.ignoreOnWindows "CliQueries" Cardano.Testnet.Test.Cli.Queries.hprop_cli_queries
           ]
         ]
-    , testGroup "SubmitApi"
-        [ testGroup "Babbage"
+    , T.testGroup "SubmitApi"
+        [ T.testGroup "Babbage"
             [ H.ignoreOnWindows "transaction" Cardano.Testnet.Test.SubmitApi.Babbage.Transaction.hprop_transaction
             ]
         ]
     ]
 
-shouldRunInParallel :: IO Bool
-shouldRunInParallel = (== Just "1") <$> E.lookupEnv "PARALLEL_TESTNETS"
+defaultMainWithIngredientsAndOptions :: [T.Ingredient] -> T.OptionSet -> T.TestTree -> IO ()
```

Comment:
Is this just copied `defaultMainWithIngredients` with ability to change `OptionSet` or is there something else added here?

Reply:
Yes, that is correct.